providers/io/engrxiv/harvester.py

```diff
@@ -16,13 +16,16 @@ class EngrxivHarvester(OSFHarvester):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.url = '{}v2/preprint_providers/engrxiv/preprints/'.format(settings.OSF_API_URL)
+        # self.url = '{}v2/preprint_providers/engrxiv/preprints/'.format(settings.OSF_API_URL)
+        # temporary harvest URL until preprint branding is complete
+        self.url = '{}v2/nodes/?filter[tags]=engrxiv'.format(settings.OSF_API_URL)
```

Comment:
OSFHarvester has `self.url = 'https://api.osf.io/v2/nodes/'`, so instead of changing it in `__init__` you could just add `url.args['filter[tags]'] = 'engrxiv'` in do_harvest down below. (It was using the staging api in my local testing, which had 0 results)